packages/three-vrm-materials-mtoon/src/nodes/MToonNodeMaterial.ts

```diff
@@ -124,6 +131,13 @@ export class MToonNodeMaterial extends Nodes.NodeMaterial {
     }
     delete parameters.transparentWithZWrite;
 
+    // `MToonMaterialLoaderPlugin` assigns these parameters to the material
+    // However, `MToonNodeMaterial` does not support these parameters
+    // so we delete them here to suppress warnings
+    delete (parameters as any).giEqualizationFactor;
+    delete (parameters as any).v0CompatShade;
+    delete (parameters as any).debugMode;
```

Comment:
Can we check `this.materialType` in `MToonMaterialLoaderPlugin` to avoid assigning at the first place?

Reply:
I have already considered that option. However, it's not preferable since `MToonMaterialLoaderPlugin.js` must import `MToonNodeMaterial` at `if (this.materialType === MToonNodeMaterial)` in that case.
I also considered using `material.isMToonNodeMaterial`, but that's an instance member instead of a static member, so it's not applicable.
Then I ended up this way.

Reply:
Can we do the check via `if (this.materialType !== MToonMaterial)` or add a flag for this if the `materialType` is not set in `MToonMaterialLoaderPluginOptions`?

Reply:
> Can we do the check via if (this.materialType !== MToonMaterial) or add a flag for this if the materialType is not set in MToonMaterialLoaderPluginOptions?

some third party material might want to use `v0CompatShade` or `debugMode`.

Reply:
Ok I see. LGTM then.